### PR TITLE
fix: Video: Rich text & links in "Additional Information"

### LINKED ISSUE #5909: Video: Enable Rich text and links in "Additional Information"
Event organizers would like to add rich text and links into the area "Additional Information". Please enable rich text in that field in the video tab and show this accordingly on the schedule.

Screenshot from 2020-12-04 18-40-47

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #5909

#### Short description of what this resolves:
Rich Text added for "Additional information" in Video tab, shows the rich text accordingly in the schedule page

SCREENSHOTS - 

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by inayahassan23** [comment]
ur branch is wrong delete it its development cant u see??

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -53,7 +53,8 @@
   </div>
   <div class="field">
     <label>{{t 'Additional Information'}}</label>
-    <Textarea
+    <Widgets::Forms::RichTextEditor
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:56]: Need to add sanitization in server before this

Comment (MeghalBisht) [reply app/components/forms/events/view/videoroom-form.hbs:56]: Not familiar with the server yet, should I close this PR?

Comment (iamareebjamal) [reply app/components/forms/events/view/videoroom-form.hbs:56]: No

**Comment by maze-runnar** [comment]
Screenshot from 2021-01-04 23-09-41
Screenshot from 2021-01-04 23-09-34

**Pull request closed without merging**